src/ncp/ncp_base.cpp

```diff
@@ -237,10 +242,15 @@ NcpBase::NcpBase(Instance *aInstance)
 #if OPENTHREAD_FTD
     , mPreferredRouteId(0)
 #endif
+    , mCurCommandIID(0)
 #if OPENTHREAD_RADIO || OPENTHREAD_CONFIG_LINK_RAW_ENABLE
     , mCurTransmitTID(0)
```

Comment:
We don't need a transmit TID for each IID. Transmit TID is for transmitting a raw packet, and it is used only while transmitting a packet to send the transmit done callback i.e. while dequeuing the packet in case of multipan feature.

Reply:
Ok. I was just thinking about future, where we may have radio able to handle two transmissions at the same time (multi-radio or some transmit queue).